pkg/trtl/mock/counterparty.go

```diff
@@ -0,0 +1,361 @@
+package mock
```

Comment:
Could we rename this file to `grpc.go` or `trtl.go` -- counterparty was specific to the RemotePeer; I'm open to other names as well.

Reply:
Does `remote.go` work?

Reply:
Works for me!